examples/pandacss/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "example-pandacss",
+  "type": "module",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Astrobook example",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "@pandacss/dev": "^0.53.6",
+    "astro": "^5.7.13",
+    "astrobook": "*"
+  },
+  "stackblitz": {
+    "installDependencies": false,
+    "startCommand": "pnpm install && pnpm panda codegen --silent && pnpm run dev"
```

```suggestion
    "startCommand": "pnpm install && pnpm run dev"
```

We don't need `pnpm panda codegen` here once we have `prepare` script